src/components/App.js

```diff
@@ -1,19 +1,49 @@
-import React from "react";
+import React, { useState } from "react";
+
 import CategoryFilter from "./CategoryFilter";
 import NewTaskForm from "./NewTaskForm";
 import TaskList from "./TaskList";
 
 import { CATEGORIES, TASKS } from "../data";
-console.log("Here's the data you're working with");
-console.log({ CATEGORIES, TASKS });
 
 function App() {
+  const [taskList, setTasks] = useState(TASKS)
+  const [categoryFilter, setCategoryFilter] = useState("All")
+  const [isSelected, setSelected] = useState("All")
+  const [formData, setFormData] = useState([])
+
+  const onTaskFormSubmit = (newFormData) => {
+    const formDataArray = [...formData, newFormData]
+    setFormData(formDataArray)
+    setTasks([...taskList, newFormData])
+  }
+
+  const deleteTasks = (event) => {
+    const currentTask = event.target.parentElement.childNodes[1].textContent
+    const updatedTaskList = taskList.filter(task => task.text !== currentTask)
+    setTasks(updatedTaskList)
+  }
```

Comment:
What do you think about making the argument a task? That was this component doesn't have to handle the event which seems more like a responsibility of the `Task` component. 
```suggestion
  const deleteTask = (taskToDelete) => {
    const updatedTaskList = taskList.filter(task => task.text !== taskToDelete)
    setTasks(updatedTaskList)
  }
```